index.html

```diff
@@ -26,21 +32,44 @@ <h2>San Francisco's first and awesome-est high school hackathon</h2>
 
 		<div id="info" class="info-tab">
 			<h1>Info</h1>
+			<center><p>San Francisco's first and best high school hackathon! More info to be posted!<p>
+			</center>
 			<p>
-			San Francisco's first and awesome-est high school hackathon! More info to be posted!
 			</p>
-			</div>
+			<p>
+			</p>
+		</div>
 		<div id="faq" class="info-tab">
 			<h1>FAQ</h1>
+			<center><p>What are we? A hackathon! More FAQs to be posted soon!</p>
+			</center>
 			<p>
-				What are we? A hackathon! More FAQs to be posted soon!
 			</p>
-			</div>
+			<p>
+			</p>
+		</div>
 		<div id="sponsors" class="info-tab">
 			<h1>Sponsors</h1>
+			<center><p>Sponsorships are currently compiling</p>
+			</center>
+			<p>
+			</p>
 			<p>
-				Sponsorships are currently compiling
 			</p>
-    </div>
+		</div>
+		<script>
+			var myIndex = 0;
```

Comment:
This old code is actually broken, but I can fix it later. It simply rotates which one is shown rather than showing the corresponding section.

Reply:
So you can move any changes that actually worked. U can pull the ico image and use it in this repo I just put it in my other repo because I wanted to see if it worked.